exporter/prometheusexporter/prometheus.go

```diff
@@ -38,6 +39,11 @@ func newPrometheusExporter(config *Config, set exporter.Settings) (*prometheusEx
 	collector := newCollector(config, set.Logger)
 	registry := prometheus.NewRegistry()
 	_ = registry.Register(collector)
+
+	if allowUTF8FeatureGate.IsEnabled() {
+		model.NameValidationScheme = model.UTF8Validation
```

Comment:
Here is the problem with making this configuration instead of a feature gate: model.NameValidationScheme is **global**.  If you made 2 exporters (or even one PRW exporter and one Prometheus exporter), and tried to enable UTF for one, and disable for the other, they would race and you would randomly get it enabled or disabled.

Otherwise, I would be totally OK with making it a config option.

Reply:
Yep, it makes a lot more sense now that I looked more into it :)

Reply:
This means that it doesn't make sense to tootle this variable in the exporter factory. Are we allowed to add things to the collector init() function?

Reply:
Feature gates used to not be initialized properly during init.  You can try it, but make sure you test it :)

Reply:
I think I'll just move the feature gate to the translation package. That way both PRW and prometheus exporter can use the same thing